django/db/models/constraints.py

```diff
@@ -65,13 +65,16 @@ def deconstruct(self):
 
 
 class UniqueConstraint(BaseConstraint):
-    def __init__(self, *, fields, name, condition=None):
+    def __init__(self, *, fields, name, condition=None, include=()):
         if not fields:
             raise ValueError('At least one field is required to define a unique constraint.')
         if not isinstance(condition, (type(None), Q)):
             raise ValueError('UniqueConstraint.condition must be a Q instance.')
+        if include and not isinstance(include, (list, tuple)):
```

Comment:
...and then this can be:
```python
if isinstance(include, (type(None), list, tuple)):
```
That is consistent with how `condition` is implemented.